pig/src/main/java/org/apache/zeppelin/pig/PigUtils.java

```diff
@@ -171,6 +171,15 @@ public static String extractFromSimplePigStats(SimplePigStats stats) {
   private static String extractFromTezPigStats(TezPigScriptStats stats) {
 
     try {
+      if (stats.getReturnCode() == PigRunner.ReturnCode.UNKNOWN) {
+        LOGGER.warn("unknown return code, can't display the results");
```

Comment:
I reread this, I guess if `getReturnCode` is returning UNKNOWN, there isn't any other meaningful code to put into log. So I'm good with the LOGGER.warn line.